getting_started_vagrantbox.md

```diff
@@ -50,7 +50,9 @@ As mentioned the virtual machine can be reached at `10.0.3.10`, meaning the serv
 
 ### CLI-tools
 Nomad, Vault and Consul have their own CLI-tools to interact with the servers that are running. They all default to `localhost`, and the default ports mentioned above. This
- means you can download any of the binaries, and they will automatically be connected to the services running inside the virtual machine. Refer to [this section](./README.md#iteration-of-development-process) to see
+ means you can download any of the binaries, and they will automatically be connected to the services running inside the virtual machine. Refer to [Iteration of The Development
```

Comment:
Not sure that I understand the correctly part, where binaries will automatically be connected after downloading.

1. Binaries already available in PATH inside vm
2. User can also download binaries on the local machines and use them

```suggestion
 means you can download any of the binaries, and use them against vagrant-hashistack. Refer to [Iteration of The Development
```

Reply:
```suggestion
 means you can download any of the binaries, and use them against the services inside the virtual machine. Refer to [Iteration of The Development
```